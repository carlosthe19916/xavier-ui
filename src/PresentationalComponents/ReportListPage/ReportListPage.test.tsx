import ReportListPage from './ReportListPage';

describe('ReportListPage', () => {
    it('expect to render ', () => {
        const wrapper = shallow(
            <ReportListPage></ReportListPage>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
