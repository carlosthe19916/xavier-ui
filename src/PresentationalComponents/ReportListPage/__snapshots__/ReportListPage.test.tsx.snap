// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportListPage expect to render  1`] = `ShallowWrapper {}`;
